Features1/tests/features/deleteContact.feature

```diff
@@ -0,0 +1,9 @@
+Feature:Delete Contact
+    As a user
+    I want to delete my contacts 
+    So that i can remove unwanted contacts
+
+    Scenario: Delete contacts
+        Given the user has navigated to the contact page
+        When  the user click "delete" botton
+        Then  the contact should be deleted
```

Comment:
and assert that the given user should be deleted. (the user should not be in the contact list anymore)